tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -2159,14 +2160,36 @@ public static void test_unsiged_integers() throws Exception {
 		conn.close();
 	}
 
+
+	public static void test_get_schema() throws Exception {
+		DuckDBConnection conn = (DuckDBConnection) DriverManager.getConnection("jdbc:duckdb:");
+
+		assertEquals(conn.getSchema(), "main");
+	}
+
+
 	public static void main(String[] args) throws Exception {
 		// Woo I can do reflection too, take this, JUnit!
 		Method[] methods = TestDuckDBJDBC.class.getMethods();
+		boolean anyFailed = false;
 		for (Method m : methods) {
 			if (m.getName().startsWith("test_")) {
 				m.invoke(null);
+				System.out.print(m.getName() + " ");
+
+				LocalDateTime start = LocalDateTime.now();
```

Comment:
This looks like debug code - should this be merged?

Reply:
I can remove the timing code, I mostly added it to show how slow some of these tests were (most are less than a second, two are over a minute each). The rest I think is useful, as long as we aren't using a real test runner such as junit or testng